.github/workflows/pre-commit.yml

```diff
@@ -28,12 +28,8 @@ jobs:
 
       - name: Install hadolint
         run: |
-          set -x
-          echo $PATH
           wget -qO /usr/local/bin/hadolint https://github.com/hadolint/hadolint/releases/latest/download/hadolint-Linux-x86_64
-          ls -l /usr/local/bin/hadolint
           chmod +x /usr/local/bin/hadolint
-          which hadolint
```

Comment:
Executing `hadolint --version`(?) might be interesting to see if it's in the PATH